mono/metadata/icall-decl.h

```diff
@@ -168,6 +168,7 @@ ICALL_EXPORT int ves_icall_System_Environment_get_Platform (void);
 ICALL_EXPORT int ves_icall_System_GC_GetCollectionCount (int);
 ICALL_EXPORT int ves_icall_System_GC_GetMaxGeneration (void);
 ICALL_EXPORT gint64 ves_icall_System_GC_GetAllocatedBytesForCurrentThread (void);
+ICALL_EXPORT gint64 ves_icall_System_GC_GetTotalAllocatedBytes (MonoBoolean precise, MonoError*);
```

Comment:
nit: I don't think you need a declaration here if you're using the `HANDLES()` form in icall-def.h